pkg/service/service.go

```diff
@@ -240,6 +240,7 @@ type Service struct {
 
 	backendRefCount counter.StringCounter
 	backendByHash   map[string]*lb.Backend
+	backendsById    map[lb.BackendID]struct{}
```

Comment:
Is this only used one for restore, if so can we null this out after the restore step is complete so it can be GCd?

Reply:
It's not used anymore. That's why I ended up adding a local variable. I'll remove the unused variable.